src/main/java/com/example/onlinebookstore/repository/CartItemRepository.java

```diff
@@ -1,9 +1,11 @@
 package com.example.onlinebookstore.repository;
 
 import com.example.onlinebookstore.model.CartItem;
+import org.springframework.data.jpa.repository.EntityGraph;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface CartItemRepository extends JpaRepository<CartItem, Long> {
+    @EntityGraph(value = "book.id")
```

Comment:
I am not sure you want to set only ID here, it is likely that you need either the whole book or nothing from it, I think ID is fetched by default, please debug and check it in accordance with what should return your endpoint that uses this method

Reply:
Remove.